test/util_test.py

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from helpers import LuigiTestCase
```

Comment:
It seems like you accidentally pulled some unrelated changes to util_test into this.